### ISSUE TITLE
Can we add this to the FreeCAD Addons Manager yet? (to replace the previous cadquery WB?)

### ISSUE THREAD

**Issue by luzpaz** [jpmlt/freecad-cadquery2-workbench#1]
https://github.com/FreeCAD/FreeCAD-addons/

**Comment by jpmlt** [comment]
@luzpaz  Yes it can be added if you think it comply with Addons Manager rules. I don't know how to make it.

**Comment by luzpaz** [comment]
@jpmlt what are the dependencies that this workbench needs? Perhaps you could create a new thread on the [FreeCAD forum](https://forum.freecad.org) so we can help migrate this in to the Addon Manager?

**Comment by luzpaz** [comment]
See https://wiki.freecadweb.org/Package_Metadata#Examples

**Comment by jpmlt** [comment]
@luzpaz thanks for the link. I will investigate it in the coming weeks, quite busy now. Where I have a doubt is that it requires to install FreeCad and CadQuery both inside a Conda environment to get this workbench working, which may not follow the Addons Manager rule.

**Comment by luzpaz** [comment]
~Conda isn't necessary~  
Do we have to use conda?
